src/LaravelDebugbar.php

```diff
@@ -481,7 +481,13 @@ public function __toString(): string
         if ($this->shouldCollect('logs', false)) {
             try {
                 $file = $config->get('debugbar.options.logs.file');
-                $this->addCollector(new LogsCollector($file));
+                if (is_string($file)) {
+                    $this->addCollector(new LogsCollector($file));
+                } else if (is_array($file)) {
+                    foreach ($file as $name => $path) {
+                        $this->addCollector(new LogsCollector($path, $name));
```

Comment:
An easy fix for that would be to just remove the `$name` arg, right?
Don't forget that loading multiple log files at the same time can be pretty resource-intensive. It might be better to split them into separate tabs than to merge all of them into the same one.

Reply:
> Don't forget that loading multiple log files at the same time can be pretty resource-intensive. It might be better to split them into separate tabs

split them into separate tabs still the same "pretty resource-intensive"